cmd/soroban-cli/src/commands/network/mod.rs

```diff
@@ -1,11 +1,9 @@
-use clap::Parser;
-
-use crate::rpc::{self};
-
 use super::{config::locator, global};
+use clap::Parser;
 
 pub mod add;
 pub mod default;
+mod health;
```

Comment:
The commands are all pub mod so that they can be embedded into other tooling. Not too many realistically have that happen, but we made them all pub so we could if needed.
```suggestion
pub mod health;
```